.github/workflows/create_wappalyzer_update.yml

```diff
@@ -0,0 +1,47 @@
+name: Create Wappalyzer Update PR
+
+on:
+  push:
+    paths:
+      README.md
```

Comment:
Just noticed now.

Reply:
Good catch, that's a leftover from the test repo. We don't need push included at all.